web/pdf_page_view.js

```diff
@@ -44,41 +44,29 @@ var TEXT_LAYER_RENDER_DELAY = 200; // ms
  */
 
 /**
- * @class
  * @implements {IRenderableView}
  */
-var PDFPageView = (function PDFPageViewClosure() {
+class PDFPageView {
   /**
-   * @constructs PDFPageView
    * @param {PDFPageViewOptions} options
    */
-  function PDFPageView(options) {
-    var container = options.container;
```

Comment:
I think it'd be a little bit clearer if we keep `let container = options.container;` here, considering that otherwise you're handling all the other parameters *except* this one at the top of the constructor.